content/applications/websites/livechat/reports.rst

```diff
@@ -0,0 +1,176 @@
+=======
+Reports
+=======
+
+Odoo *Live Chat* includes several reports that allow for the monitoring of operator performance and
+the identification of trends in customer conversations.
+
+Available reports
+=================
+
+The following reports are included in the *Live Chat* app:
+
+- :ref:`Sessions History <livechat/sessions-history>`
+- :ref:`Session Statistics <livechat/session-statistics>`
+- :ref:`Operator Analysis <livechat/operator-analysis>`
+
+.. note::
+   The *Live Chat Ratings Report* can also be accessed through the :guilabel:`Report` menu. For more
+   information on this report, and on the *Live Chat* rating process, see :doc:`Live Chat Ratings
+   <../livechat/ratings>`.
+
+To access the available reports, navigate to the :menuselection:`Live Chat app --> Report menu`.
+
+.. _livechat/sessions-history:
+
+Sessions History
+----------------
+
+The *Sessions History* report displays an overview of live chat sessions, including participant
+names, session dates, and the duration of the session. It also provides access to the complete
+transcripts of live chat sessions, which open in the *Discuss* app.
+
+.. image:: reports/sessions-history.png
```

Comment:
For some reason it give me an "unable to render" error for this image below. I don't want to reload the page in case I lose my comments, so I'd check that it's compressed and/or appears. Could totally be a guthub error on my end.